### ISSUE TITLE
Option to indicate on a curve the epochs where learning rate value changed

### ISSUE THREAD

**Issue by ismael-elatifi** [tensorflow/tensorboard#3669]
We often look both at our metric curve and the LR curve to see if an improvement on our metric value happened just after a change in the LR value.
It would be nice if we could have an option to automatically show the epochs at which the LR changed on any chosen curve.
"Indication" could be just a vertical line on the plot at each epoch where the learning rate changed.
It could be implemented with a checkbox under a curve to activate/deactivate LR change indications and a dropdown list to choose the name of the LR tag (so that it would work whatever the LR tag is). 
This is particularly useful with ReduceLROnPlateau callback where the LR changes are rare.

**Comment by bmd3k** [comment]
Thanks Ismael,

We discussed briefly as a team. We think this is an interesting use case. We may try to find a more generalizable solution to this problem.  We'll put it on our backlog!

**Comment by nfelt** [comment]
FWIW, related to #411 which is a request for being able to mark timestamps/epochs in a somewhat more generic way.

**Comment by SBCV** [comment]
Are there any updates on this issue?
